recipe/meta.yaml

```diff
@@ -12,13 +12,17 @@ source:
     - CMakeLists.txt.patch  # [win]
     - config.h.patch  # [win]
     - configure.cmake.patch  # [win]
+    - utf_8_mac.patch  # [osx]
```

Comment:
```suggestion
    # MRB: I made this patch by diffing libiconv-1.17 and the version of the code here 
    # https://github.com/fumiyas/libiconv-utf8mac/commit/2677394480f0d67d6f64b74045b9b4ad08097592
    # I removed a bunch of the extraneous changes to just pull in the header fixes
    - utf_8_mac.patch  # [osx]
```

Reply:
Ah, didn't realize that.
I'd prefer we used upstream sources from Apple to be more consistent to what Homebrew/MacPorts do.
Looking at https://github.com/fumiyas/libiconv-utf8mac/commits/utf-8-mac-51.200.6.libiconv-1.17/lib/utf8mac.h the changes on top of Apple's are mostly to build on other platforms but also to support surrogate pairs -- not bad additions actually, but also most people would run Apple/Homebrew/MacPorts builds on macOS..

Reply:
Ok. I can try to rework things later.

Reply:
@ocefpaf, we can merge once this is addresses.
@beckermr, I'll see to update the patch and push it here in bit.